pipeline/patch_VAE.py

```diff
@@ -48,7 +49,8 @@ def extract_patches(paths):
             process_site_extract_patches(site_path, 
                                          site_segmentation_path, 
                                          site_supp_files_folder,
-                                         window_size=256)
+                                         window_size=256,
+                                         save_fig=True)
```

Comment:
I exposed this flag in the latest PR, together with a couple of other keyword arguments.